Makefile

```diff
@@ -5,12 +5,57 @@ Minversion := $(shell date)
 BUILD_NODE_PAR = -ldflags "-X github.com/ontio/ontology/common/config.Version=$(VERSION)" #-race
 BUILD_NODECTL_PAR = -ldflags "-X main.Version=$(VERSION)"
 
-all:
+ARCH=$(shell uname -m)
+DBUILD=docker build
+DRUN=docker run
+DOCKER_NS ?= ontio
+DOCKER_TAG=$(ARCH)-$(VERSION)
+ONT_CFG_IN_DOCKER=config-solo.json
+WALLET_FILE=wallet.dat
+
+all: ontology nodectl
+
+ontology:
```

Comment:
Is there a step need to glide install.

Reply:
"glide install" still need to be done by user after 'git clone'
As "make all", "make docker" also just build ontology for docker.